packages/node_modules/@ciscospark/widget-recents/src/components/space-item/index.js

```diff
@@ -40,7 +41,7 @@ export default function SpaceItem({ // eslint-disable-line complexity
 
 
   function handleCallClick(e) {
-    if (type === `direct`) {
+    if (type === `direct` || hasGroupCalling) {
```

Comment:
What do you think about moving this check to a higher level. I don't think the `SpaceItem` component should determine if it should have a button or not, but maybe a prop like `hasCallingSupport` passed down from the parent.

Reply:
Yea, we're going to need some sort of space status object for each space in the spaces list, especially once the "call in progress" feature is added. Definitely something to think about.